pkgs/by-name/no/noita-entangled-worlds/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, pkgs }:
+
+pkgs.stdenv.mkDerivation rec {
+  pname = "noita_entangled_worlds";
+  version = "0.32.5";
+
+  # Fetch the correct Linux zip file
+  src = pkgs.fetchurl {
+    url = "https://github.com/IntQuant/noita_entangled_worlds/releases/download/v${version}/noita-proxy-linux.zip";
+    sha256 = "sha256-Qx25tvmq/0NDvDnnqw+8HoQP5uzoMXKsK2PsiVntW+4=";
+  };
+
+  # Include unzip and makeWrapper in the build environment
+  nativeBuildInputs = [
+    pkgs.unzip
+    pkgs.makeWrapper
+    pkgs.steam-run
```

Comment:
`steam-run` should be a `buildInput` because you aren't invoking it during the build. It needs to match the architecture of the package host, not the build machine.